contracts/controller/src/views.rs

```diff
@@ -67,11 +67,44 @@ impl Contract {
     pub fn view_prices(&self, dtokens: Vec<AccountId>) -> HashMap<AccountId, Price> {
         self.get_prices_for_dtokens(dtokens)
     }
+
+    pub fn get_price_by_ticker(&self, ticker: String) -> u128 {
```

Comment:
get_price_by_ticker should be placed at the same place as get_prices_for_dtokens method does.

Reply:
At the same time you method called: get_price_by_ticker
So my expectations to see Price struct as a result
It's better to use filter with market.ticker_id == ticker condition